specs/decisionmatrix/azure.yaml

```diff
@@ -219,222 +219,24 @@ rows:
           priority: 3
           thin_provisioning: false
           drive_type: "Standard_LRS"
-        - min_iops: 120
-          max_iops: 120
+        - min_iops: 19000
```

Comment:
Why is this 19000?
Per math here
https://learn.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssd-v2-iops
formula will be
((size - 6) * 500) + 3000
((32-6) * 500) + 3000
= 16000.  
What am I missing?

Reply:
**Now that I read it again, I feel minimum should be 3000 iops** . (All Premium SSD v2 disks have a baseline IOPS of 3000 that is free of charge. After 6 GiB, the maximum IOPS a disk can have increases at a rate of 500 per GiB, up to 80,000 IOPS. So an 8 GiB disk can have up to 4,000 IOPS, and a 10 GiB can have up to 5,000 IOPS. To be able to set 80,000 IOPS on a disk, that disk must have at least 160 GiBs. Increasing your IOPS beyond 3000 increases the price of your disk)
@nrevanna for a 32 gb disk - minimum can still be 3000 and maximum can be 16000.
should I go with 3000 ??

Reply:
@nrevanna can you also help me with the priority here ?? If you know on what basis this is set ?

Reply:
Priority is not used today. You can keep the same value as other disk type entries
3000 makes sense. Let's keep that for min iops.

Reply:
fixed.